.github/workflow_scripts/run_benchmark.sh

```diff
@@ -32,7 +34,10 @@ if [ $BRANCH_OR_PR_NUMBER != "master" ]; then
     fi
 fi
 
-python CI/bench/evaluate.py --config_path ./ag_bench_runs/$MODULE/ --module_name $MODULE --time_limit $TIME_LIMIT --branch_name $BRANCH_OR_PR_NUMBER
+python CI/bench/evaluate.py --config_path ./ag_bench_runs/$MODULE/ --module_name $MODULE --time_limit $TIME_LIMIT --branch_name $BRANCH_OR_PR_NUMBER --benchmark_type $BENCHMARK
+
+echo "Deleting version1.0_file"
+rm -f ./results/version1.0*
```

Comment:
Why these logics are needed?

Reply:
This is needed for report generation, we download the version 1.0 cleaned benchmark file and evaluate the current cleaned benchmark run against this file, and then we delete the version1.0 file.
We then copy the current cleaned file to S3, thus we do not need the version1.0 file.